Sources/GRPC/ClientConnection.swift

```diff
@@ -324,6 +330,7 @@ extension Channel {
   }
 
   func configureGRPCClient(
+    httpTargetWindowSize: Int = 65535,
```

Comment:
I don't think we need a default here: we should have always have a value from the configuration.